tests/nn/data_parallel/test_sharded_ddp.py

```diff
@@ -133,67 +133,66 @@ def run_ddp_parity(rank, world_size, backend, temp_file_name):
     torch.cuda.set_device(rank)
```

Comment:
@min-xu-ai checking with you, the ddp_parity test runs with cuda_count(), so it should be 4 GPUs on CI. Is that ok ? (sanity checking that I'm not missing anything)

Reply:
I don't know CI's gpu limit. If it is >2, then it is great. Also, I have seen bugs only shows up in gpu>5 in other cases. (not for oss or shardedDP).

Reply:
ouch for 5+, I'll test that on fb cluster again. Seems like CI/unit tests is 4 gpus (https://app.circleci.com/pipelines/github/facebookresearch/fairscale/1644/workflows/9e1b0fc9-92dd-4d4e-96de-be775cf5634b/jobs/7982)

Reply:
checking with f250620664 [FB only]